src/Locales/nl/Pode.psd1

```diff
@@ -0,0 +1,286 @@
+@{
```

Comment:
A lot of this file looks to be in English, with only some translations in Dutch, is that right?

Reply:
Yes you are right let me update this list

Reply:
I don't understand the reason, but ChatGPT was reluctant to translate the entire file